portal-services/src/main/resources/portal-services-openapi.yaml

```diff
@@ -11112,6 +11112,62 @@ paths:
                 $ref: '#/components/schemas/Client'
         500:
           $ref: '#/components/responses/GenericApiError'
+          
+  /portal/client/forCustomer:  
+    get:
+      tags:
+      - Clients
+      summary: Get list of clients for a given customer by equipment ids
+      operationId: getForCustomer
+      parameters:
+      - name: customerId
+        in: query
+        description: Customer ID
+        required: true
+        schema:
+          type: integer
+          format: int32
+      - name: equipmentIds
+        in: query
+        description: Equipment ID
+        required: false
+        content:
+          text/plain:
+            schema:
+              type: array
+              items:
+                type: integer
+                format: long
```

Comment:
please use "format: int64" when describing java long datatypes.

For the reference - look at https://swagger.io/specification/ - section "Data Types"